src/test/java/teammates/test/cases/ui/pagedata/InstructorCourseEditPageDataTest.java

```diff
@@ -0,0 +1,80 @@
+        instructorList.add(dataBundle.instructors.get("instructor1OfCourse1"));
+        instructorList.add(dataBundle.instructors.get("instructor2OfCourse1"));
+        instructorList.add(dataBundle.instructors.get("helperOfCourse1"));
+        
+        InstructorAttributes currentInstructor = dataBundle.instructors.get("instructor1OfCourse1");
+        
+        int offset = -1;
+        
+        List<String> sectionNames = new ArrayList<String>();
+        sectionNames.add("Section 1");
+        sectionNames.add("Section 2");
+        
+        List<String> feedbackSessionNames = new ArrayList<String>();
+        feedbackSessionNames.add("First feedback session");
+        feedbackSessionNames.add("Second feedback session");
+        feedbackSessionNames.add("Grace Period Session");
+        feedbackSessionNames.add("Closed Session");
+        feedbackSessionNames.add("Empty session");
+        feedbackSessionNames.add("non visible session");
+        
+        InstructorCourseEditPageData pageData = new InstructorCourseEditPageData(account, course, 
+                                                                                 instructorList, 
+                                                                                 currentInstructor, 
+                                                                                 offset, sectionNames, 
+                                                                                 feedbackSessionNames);
+        
+        assertNotNull(pageData.getDeleteCourseButton());
+        assertNotNull(pageData.getAddInstructorButton());
+        
+        assertNotNull(pageData.getInstructorPanelList());
+        assertEquals(instructorList.size(), pageData.getInstructorPanelList().size());
+        assertEquals(4, pageData.getInstructorPanelList().get(0).getPermissionInputGroup1().size());
+        assertEquals(4, pageData.getInstructorPanelList().get(0).getPermissionInputGroup2().size());
+        assertEquals(3, pageData.getInstructorPanelList().get(0).getPermissionInputGroup3().size());
+        assertEquals(sectionNames.size(), pageData.getInstructorPanelList().get(0).getSectionRows().size());
+        assertEquals(feedbackSessionNames.size(), pageData.getInstructorPanelList().get(0).getSectionRows().get(0).getFeedbackSessions().size());
+        
+        assertNotNull(pageData.getAddInstructorPanel());
+        assertEquals(4, pageData.getAddInstructorPanel().getPermissionInputGroup1().size());
+        assertEquals(4, pageData.getAddInstructorPanel().getPermissionInputGroup2().size());
+        assertEquals(3, pageData.getAddInstructorPanel().getPermissionInputGroup3().size());
+        assertEquals(sectionNames.size(), pageData.getAddInstructorPanel().getSectionRows().size());
+        assertEquals(feedbackSessionNames.size(), pageData.getAddInstructorPanel().getSectionRows().get(0).getFeedbackSessions().size());
+        
```

Comment:
Is this sufficient?
It seems to miss the instructor remind button branch entirely.